packages/ses/test/test-freeze-module-namespace.js

```diff
@@ -0,0 +1,22 @@
+// import '../index.js';
+// import './lockdown-safe.js';
+import test from 'ava';
+
+import * as namespace from './module-namespace-const-1.js';
+
+const { freeze, isFrozen, isExtensible, getOwnPropertyDescriptors } = Object;
+
+test('freeze module-namespace-const-1', t => {
+  t.is(isFrozen(namespace), false, 'not pre-frozen');
+  t.is(isExtensible(namespace), false, 'is pre non-extensible');
+  const preDescs = getOwnPropertyDescriptors(namespace);
+  t.is(preDescs.one.writable, true, 'export const is unfortunately writable');
+  t.is(preDescs.one.configurable, false, 'at least it is not configurable');
+  t.throws(() => freeze(namespace), {
+    message: /Cannot assign to read only property 'one' of object '\[object Module\]'/,
+  });
+  t.is(isFrozen(namespace), true, 'now it is frozen, really?');
```

Comment:
That sounds like bullshit.

Reply:
Just to be clear, you are criticizing the observed Node behavior, rather than the test code?

But if the test code demonstrates a spec non-conformance, which I believe it does, then we have at least the following TODOs:
   * TODO file a bug against at least v8, where this behavior was observed.
   * TODO test the other major engines, filing bugs appropriately.
   * TODO use the Ava API to rephrase the test case as testing that something bad still happens, so we can find out if the bad behavior gets fixed.

The last is something that should be done in this PR.

Reply:
> Just to be clear, you are criticizing the observed Node behavior

Yes I am! In hindsight that was not very clear, sorry.

> * file a bug against at least v8

I believe we did and it was fixed right away. Not sure which node version the fix is included in, but I can try to dig that out tomorrow